### DISCUSSION TITLE
Error ketika menggunakan explain difungsi Lime utk interprestasi klasifikasi (Capstone ML)

### DISCUSSION THREAD

**Discussion by Micoalgo** [teamalgoritma/community#640] [Q&A] [answered]
Ketika menjalankan fungsi explain di Lime menemukan error yaitu sbb ;

<img width="534" alt="image" src="https://user-images.githubusercontent.com/123436963/232335638-3d07d6ec-a21d-417c-be6b-236ea4aaf3bd.png">

Bagaimanakah memperbaikinya ? 
Terimakasih

**Comment by amaliasf** [comment, marked as answer]
Dear Pak @Micoalgo 

Terimakasih atas pertanyaannya Pak. Untuk kendala yang Bapak hadapi sepertinya serupa dengan diskusi https://github.com/teamalgoritma/community/discussions/486 

Boleh silahkan dicek Pak, semoga membantu ^^

**Reply by Micoalgo** [reply]
Terimakasih jawabannya Mba Amalia
